eclcli/common/utils.py

```diff
@@ -339,9 +342,12 @@ def parse_allocation_pools(text):
         'end': lst[1]
     }
 
+
 def validate_ipv4(text):
     try:
-        ipaddress.IPv4Address(unicode(text))
+        if type(text) is not six.text_type:
+            text = six.u(text)
```

Comment:
Skip processing if `text` is already unicode type to avoid raising exception from `six.u` .